4337/certora/scripts/verify.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+params=("--send_only")
+
+if [[ -n "$CI" ]]; then
+    params=()
+fi
+
+certoraRun certora/conf/Safe4337Module.conf \
+    "${params[@]}" \
+    --msg "Safe4337Module $*" \
+    "$@"
+
+# certoraRun certora/conf/PayForMissingFunds.conf \
+#     "${params[@]}" \
+#     --msg "Safe4337Module $*" \
+#     "$@"
+
+# certoraRun certora/conf/ExecTransactionFromModule.conf \
+#     "${params[@]}" \
+#     --msg "Safe4337Module $*" \
+#     "$@"
+
+# certoraRun certora/conf/ValidationDataLastBitOne.conf \
+#     "${params[@]}" \
+#     --msg "Safe4337Module $*" \
+#     "$@"
```

Comment:
Should these be commented out?

Reply:
The PR description says that they're wip, so I guess so. Would be good to add a comment tho